### PR TITLE
Projects by owner page

### PR DESCRIPTION
**Pull request by patrickkerrypei**

### CODE REVIEW

File: src/common/containers/content/pages/widgets/data_tables/UsersTable.jsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+// Self-defined
+import DataTable from '../../../../../components/content/widgets/data_tables/DataTable';
+import UsersDataTableEntry from './table_entries/UsersDataTableEntry';
+import { sortWithChecks } from '../../../../../../client/utils/utils';
+import { fetchUsersIfNeeded } from '../../../../../actions/users';
+import { sortBy } from '../../../../../actions/tables';
+
+const USERS_FIELDS = {
```
Comment (pmeijer) [inline src/common/containers/content/pages/widgets/data_tables/UsersTable.jsx]: Is this right for the UsersTable?

File: src/common/containers/content/pages/widgets/data_tables/table_entries/ProjectsDataTableEntry.jsx [hunk 2]
```diff
@@ -9,8 +9,12 @@ import { connect } from 'react-redux';
 import ProjectsDataTableEntry from '../../../../../../components/content/widgets/data_tables/table_entries/ProjectsDataTableEntry'; // eslint-disable-line max-len
 
 const mapStateToProps = (state) => {
+    const { basePath } = state;
+    const { user } = state.user;
```
Comment (pmeijer) [inline src/common/containers/content/pages/widgets/data_tables/table_entries/ProjectsDataTableEntry.jsx]: Is the guarded against the user not being loaded at this time?

Comment (patrickkerrypei) [inline src/common/containers/content/pages/widgets/data_tables/table_entries/ProjectsDataTableEntry.jsx]: I think it'll just pull the initial user state from the [default reducer state](https://github.com/webgme/user-management-page/blob/projects-by-owner-page/src/common/reducers/users.js#L8)

File: src/common/containers/content/pages/widgets/data_tables/UsersTable.jsx (hunk 1 above)
Comment (patrickkerrypei) [inline src/common/containers/content/pages/widgets/data_tables/UsersTable.jsx]: That's wrong - I'll update it on this branch

File: src/common/containers/content/pages/widgets/data_tables/table_entries/ProjectsDataTableEntry.jsx (hunk 2 above)
Comment (pmeijer) [inline src/common/containers/content/pages/widgets/data_tables/table_entries/ProjectsDataTableEntry.jsx]: Your link pointed to users.
What I'm getting at is if you're guarding against issues when the "state.user" is the initial value.

Comment (patrickkerrypei) [inline src/common/containers/content/pages/widgets/data_tables/table_entries/ProjectsDataTableEntry.jsx]: Oh I see, I'll double check all the uses (for the other initial values too)

**Pull request merged**